lib/DDG/Spice/Crunchbase/Organizations.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Crunchbase::Organizations;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/spice_overview if you are new
+# to instant answer development
+
+use DDG::Spice;
+
+use strict;
+
+# Triggers
+triggers startend => '///***never trigger***///';
+spice to => 'http://api.crunchbase.com/v/2/organization/$1?user_key={{ENV{DDG_SPICE_CRUNCHBASE_APIKEY}}}';
```

Comment:
Same as in https://github.com/duckduckgo/zeroclickinfo-spice/pull/1984/files#r33839657